bio.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get To Know Me</title>
+    <link rel="stylesheet" href="bio.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+<p style="background-image: url('https://media.istockphoto.com/id/484470418/photo/soap-bubbles-on-a-white-background.jpg?b=1&s=170667a&w=0&k=20&c=45-k41wIQcIQ-iuQNEdYU0W9yUTs7lN0Z0tc0u5a6b0=');"></p>
```

Comment:
avoid putting non-metadata tags in the header - the <head> should only have tags such as meta, link, title, etc.